packages/rocketchat-lib/server/methods/getChannelHistory.js

```diff
@@ -0,0 +1,90 @@
+Meteor.methods({
+	getChannelHistory(rid, latest, oldest, inclusive, count, unreads) {
```

Comment:
What do you think about using `{rid, latest, oldest, inclusive, count, unreads}` instead of `rid, latest, oldest, inclusive, count, unreads` so who call can pass named parameters.

I think named parameters as a better option for APIs and for readability.